.github/workflows/unit-tests.yml

```diff
@@ -220,6 +221,27 @@ jobs:
           printf 'deviceType=IpStb\n' | sudo tee /etc/authService.conf
           printf 'imagename:PX051AEI_VBN_2203_sprint_20220331225312sdy_NG\nSDK_VERSION=17.3\nMEDIARITE=8.3.53\nYOCTO_VERSION=dunfell\n' | sudo tee /version.txt
 
+          sudo mkdir /opt/secure
+          sudo mkdir /opt/persistent
+          sudo mkdir /opt/secure/reboot
+          sudo mkdir /opt/secure/persistent
+          sudo mkdir /opt/secure/persistent/System
+          sudo mkdir /opt/logs
+          sudo mkdir /lib/rdk
+          sudo chmod -R 1777 /opt
+          sudo chmod -R 1777 /lib/rdk
+
+          printf 'PreviousRebootReason: 18.08.2022_09:51.38 RebootReason: Triggered from SystemServices! MAINTENANCE_REBOOT \n PreviousRebootTime: 18.08.2022_09:51.38 \n PreviousCustomReason:  MAINTENANCE_REBOOT \n PreviousOtherReason:  MAINTENANCE_REBOOT \n' |  sudo tee /opt/logs/rebootInfo.log
+          printf '{\n\"timestamp\":\"Thu Aug 18 13:51:39 UTC 2022\",\n \"source\":\"HAL_CDL_notify_mgr_event",\n \"reason\":\"OPS_TRIGGERED\",\n \"customReason\":\"Unknown",\n \"otherReason\":\"Rebooting the box due to VL_CDL_MANAGER_EVENT_REBOOT...!\"\n}\n' |  sudo tee /opt/persistent/previousreboot.info
+          printf '{\n\"lastHardPowerReset\":\"Thu Aug 18 13:51:39 UTC 2022\"\n}\n' |  sudo tee /opt/persistent/hardpower.info
+          printf 'RDK_STARTED:35755' | sudo tee /opt/logs/rdk_milestones.log
+          printf 'false' | sudo tee /opt/tmtryoptout
+          printf '#!/bin/sh' | sudo tee /lib/rdk/getStateDetails.sh
```

Comment:
please mock system instead

Reply:
Supporting unit test will added in the next set, so removed the file updates